src/pages/jobs.tsx

```diff
@@ -15,7 +15,7 @@ const IndexPage = () => {
     <Layout>
       <SEO
         description={
-          'Build scalable open source software at one of the fastest growing startups in the space of cloud identity'
+          'Build scalable open source software at Ory, a fast growing startup dedicated to security, identity and authorisation in the cloud.'
```

Comment:
```suggestion
          'Build scalable open source software at Ory, a fast-growing startup dedicated to security, identity, and authorization in the cloud.'
```

We generally use the North American spelling in all texts.